### ISSUE TITLE
Elementor Atomic Elements widgets: contrast failures reported as incomplete, not saved as issues

### ISSUE THREAD

**Issue by pattonwebz** [equalizedigital/accessibility-checker#1942]
## What

Follow-up from #1938. That PR fixes false positives from Elementor's editor UI during front-end highlighter rescans, and hides `.elementor-element-overlay` (plus `.edac-highlight-btn`) around `axe.run` so contrast checks aren't obscured by the legacy widget overlay. That part works correctly.

However, testing against Elementor's newer **Atomic Elements** widget system (Elementor 4.1.x, e.g. the `e-heading` widget) turned up a case the fix doesn't cover: a genuinely failing-contrast widget is reported by axe as `incomplete` (undecidable) rather than as a violation, so it never appears in the saved issue list after a rescan. This is a false negative — a real accessibility bug goes unreported.

## Repro

1. In Elementor 4.1.x, add a Heading widget (Atomic Elements) to a page.
2. Set its text color to a near-white value (e.g. `#F5F5F5`) with a white background — a clear WCAG AA contrast failure (~1.05:1 ratio).
3. Save the page and use the accessibility-checker front-end highlighter's "Rescan This Page" inside the Elementor editor.
4. The issue count does not include a contrast violation for the heading, even though the text is visibly near-invisible on screen.

## Root cause (confirmed via direct axe-core inspection)

Instrumenting `axe.run` inside the Elementor preview iframe and inspecting the raw per-rule results for `color_contrast_failure` showed:

```json
{
  "violations": 0,
  "incomplete": 1,
  "incompleteDetail": {
    "any": [{
      "id": "color-contrast",
      "message": "Element's background color could not be determined because it's partially obscured by another element",
      "data": { "messageKey": "elmPartiallyObscured", "contrastRatio": 0, "expectedContrastRatio": "3:1" }
    }],
    "node": {
      "selector": [".e-27fea3b-51fa0fc"],
      "source": "<h2 data-interaction-id=\"27fea3b\" class=\"e-27fea3b-51fa0fc e-heading-base\" draggable=\"true\">EDAC test low contrast heading</h2>"
    }
  }
}
```

This is the exact `elmPartiallyObscured` occlusion bug class #1938 fixes — but for a widget with **no** `.elementor-element-overlay` child element at all (confirmed via DOM inspection: the atomic `e-heading` widget wrapper's only child is the `<h2>` itself). Elementor's Atomic Elements editor UI uses a different mechanism to indicate the widget in edit mode than the legacy `.elementor-element-overlay` div the current `overlayHideSelectors` list targets, so `hideOverlaysDuringScan()` has nothing to hide for these widgets and the occlusion persists.

## Scope note

#1938's PR description says it was verified against Elementor 3.18.3 (legacy widget rendering). This gap only shows up on newer Elementor versions using Atomic Elements, so it's reasonable this wasn't caught in that PR's testing — filing separately rather than blocking #1938 on it.

## Suggested next step

Identify what actually occludes an Atomic Elements widget in edit mode (it isn't a sibling `.elementor-element-overlay` div) and extend `overlayHideSelectors` in `src/pageScanner/config/exclusions.js` to cover it, or find an equivalent hide-during-scan mechanism for the atomic widget editor chrome.

**Comment by pattonwebz** [comment]
Confirmed false positive — closing this out.

Re-tested the repro with a clean environment across Elementor 4.1.3 and 4.3.0 (updated to latest to double-check), covering the manual "Rescan This Page" flow (widget deselected and selected), a fully discarded-and-reloaded page, and the exact original flow (add widget, set failing contrast color, click Publish directly). Every run correctly reported the low-contrast Atomic Heading widget as a `violation`, never `incomplete` — #1938's `hideOverlaysDuringScan()` handles it correctly.

Direct `axe.run` inspection during the original investigation also turned up a stale `.edac-highlight-btn` marker (this plugin's own leftover issue-marker from a prior scan) sitting on top of the widget, and confirmed #1938's fix hides it properly during a real scan. Best read: this was contamination from leftover state in my own manual test session, not a real gap in #1938. Retracted the corresponding PR comment and approved #1938.
